Dockerfile

```diff
@@ -16,7 +16,11 @@ LABEL maintainer="Nick Badger <nbadger@mintel.com>" \
       org.opencontainers.image.revision="$VCS_REF"
 
 COPY --from=banzaicloud/vault-env:1.3.2 /usr/local/bin/vault-env /usr/local/bin/
-COPY --from=prom/mysqld-exporter:v0.12.1 /bin/mysqld_exporter /bin
+
+# Temp copy fow since it fixes a bug in dsn handling.
+# https://github.com/prometheus/mysqld_exporter/pull/485
+# COPY --from=prom/mysqld-exporter:v0.12.1 /bin/mysqld_exporter /bin
+COPY --from=nabadger/mysql-exporter:0.12.1_mintel-fix-dsn /bin/mysqld_exporter /bin
```

Comment:
shoud this be on `mintel/` ?

Reply:
I don't know, it's a temporary fix that I did on my own account, that is basically a branch anyway. I'm expecting them to merge the PR in, at which we point we can revert to the normal upstream releases.

Maybe if they don't merge it in in 30d I can move it to mintel?

Reply:
Given that we have to update OPA rules for this to work, I may as well copy it to mintel too....

Reply:
cool